app/Filament/Shared/WithXlsformModuleVersionQuestionEditing.php

```diff
@@ -73,7 +73,7 @@ private function customModuleQuestionTable(Table $table, Collection $locales, Xl
                 ])
                 ->schema([
                     Hidden::make('row_number')
-                        ->default(function () {
```

Comment:
I'm pretty certain the $xlsformModuleVersion here is intended as the one passed into the `customModuleQuestionTable()`, so I would do something like `->default(function () use ($xlsformModuleVersion) {...}`, or make it an inline arrow function.